src/UKK6.yaml

```diff
@@ -0,0 +1,43 @@
+---
+- name: Syntax character edge cases
+  from: '@ingydotnet'
+  tags: edge
```

Comment:
```suggestion
  tags: edge empty-key
```

Reply:
This is not correct at the moment, because all subtests get the `empty-key` tag.
The data-symlinks script needs to be fixed.

Reply:
https://github.com/yaml/yaml-test-suite/issues/107